### ISSUE TITLE
broken example player

### ISSUE THREAD

**Issue by cyberpunkbln** [parasyte/sonant-rs#12]
Hello,
when i try to compile i get this error:
```

 C:\Mounts\externalDrives\extHDD\Development\Rust\Users\rust_user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\cpal-0.11.0\src\host\wasapi\stream.rs:844:25
    |
844 |             winapi::um::errhandlingapi::GetLastError()
    |                         ^^^^^^^^^^^^^^ could not find `errhandlingapi` in `um`
    |
note: found an item that was configured out
   --> C:\Mounts\externalDrives\extHDD\Development\Rust\Users\rust_user\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.3.9\src\um\mod.rs:113:44
    |
113 | #[cfg(feature = "errhandlingapi")] pub mod errhandlingapi;
    |                                            ^^^^^^^^^^^^^^
    = note: the item is gated behind the `errhandlingapi` feature
```

give it an better example for an player or embbed play?

thx

**Comment by parasyte** [comment]
Thanks for letting me know! I've been working on updating all of the dependencies to fix this. It might take a little while before it's all working.

**Comment by parasyte** [comment]
@cyberpunkbln I just published version 0.2.0 to fix this!

**Comment by cyberpunkbln** [comment]
thx
